### PR TITLE
CHANGELOG.md: Add 2.1.10

### PR DESCRIPTION
**Pull request by july2993**
### What problem does this PR solve? 
CHANGELOG.md: Add 2.1.10

### Check List 

Tests 

 - No code

Code changes

 Side effects

 Related changes

### CODE REVIEW

**Comment by july2993** [comment]
/run-all-tests tidb=release-2.1 tikv=release-2.1 pd=release-2.1

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # TiDB-Binlog
 All notable changes to this project are documented in this file.
 
+## [2.1.10]
+- Fix all issue reported by static linters of golangci-lint [#599](https://github.com/pingcap/tidb-binlog/pull/599)
+- Drainer: Ignore empty string when getting compressor config [#598](https://github.com/pingcap/tidb-binlog/pull/598)
+- Update parser dependency [#598](https://github.com/pingcap/tidb-binlog/pull/598)
```
Comment (suzaku) [inline CHANGELOG.md]: Shall we document the specific version change here?

Comment (july2993) [reply CHANGELOG.md]: updated

**Comment by july2993** [comment]
/run-all-tests tidb=release-2.1 tikv=release-2.1 pd=release-2.1

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
 # TiDB-Binlog
 All notable changes to this project are documented in this file.
 
+## [2.1.10]
+- Fix all issue reported by static linters of golangci-lint [#599](https://github.com/pingcap/tidb-binlog/pull/599)
+- Drainer: Ignore empty string when getting compressor config [#598](https://github.com/pingcap/tidb-binlog/pull/598)
+- Update parser dependency [version](https://github.com/pingcap/parser/tree/361d2d4f774d779e3292d48af37bdce8f4ca88bf) as TiDB use  [#608](https://github.com/pingcap/tidb-binlog/pull/608)
```
Comment (suzaku) [inline CHANGELOG.md]:
```suggestion
- Update [parser](https://github.com/pingcap/parser) to the same [version](https://github.com/pingcap/parser/tree/361d2d4f774d779e3292d48af37bdce8f4ca88bf) as used in TiDB  [#608](https://github.com/pingcap/tidb-binlog/pull/608)
```

**Comment by july2993** [comment]
/run-all-tests tidb=release-2.1 tikv=release-2.1 pd=release-2.1

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,12 @@
 # TiDB-Binlog
 All notable changes to this project are documented in this file.
 
+## [2.1.10]
+- Fix all issue reported by static linters of golangci-lint [#599](https://github.com/pingcap/tidb-binlog/pull/599)
```
Comment (amyangfei) [inline CHANGELOG.md]:
```suggestion
- Fix all issues reported by static linters of golangci-lint [#599](https://github.com/pingcap/tidb-binlog/pull/599)
```

**Comment by july2993** [comment]
/run-all-tests tidb=release-2.1 tikv=release-2.1 pd=release-2.1

**Pull request merged**